### PR TITLE
Update linux.md

### PR DESCRIPTION
**Pull request by kkrum**
- Add a known issues section  that we'll use for the next few months as we iron out problems
- Add info around using always connect vs. connect.

### CODE REVIEW

File: products/warp-client/src/content/setting-up/linux.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ order: 6
 
 # Linux desktop client
 
+## Known Issues
+Thank you for using the intial release of the Linux client for WARP! Here is a list of known issues. Please check back frequently as we keep this updated.
```
Comment (abracchi-tw) [inline products/warp-client/src/content/setting-up/linux.md:9]:
```suggestion
Thank you for using the initial release of the Linux client for WARP! In this section we will call out known issues. Please check back frequently as we keep this list updated.
```

File: products/warp-client/src/content/setting-up/linux.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ order: 6
...
+* Cloudflare for Teams is not yet fully supported
```
Comment (abracchi-tw) [inline products/warp-client/src/content/setting-up/linux.md:10]: Do we need to call out what is and what isn't supported? The word `fully` makes me wonder

File: products/warp-client/src/content/setting-up/linux.md [hunk 3]
```diff
@@ -21,6 +27,10 @@ To connect for the very first time you must call `register` first:
 1. Connect `warp-cli connect`
 1. Run `curl https://www.cloudflare.com/cdn-cgi/trace/` and verify that `warp=on`
 
+### Always stay connected
+If you want to always stay connected to WARP you must call `enable-always-on`. This is the cli equivilent to switching the toggle switch to the on position in our GUI apps.
```
Comment (abracchi-tw) [inline products/warp-client/src/content/setting-up/linux.md:31]:
```suggestion
If you want WARP to be constantly enabled, call `enable-always-on`. This is the CLI equivalent to switching the toggle on in our UI.
```

File: products/warp-client/src/content/setting-up/linux.md [hunk 4]
```diff
@@ -21,6 +27,10 @@ To connect for the very first time you must call `register` first:
...
+1. Run `warp-cli enable-always-on`
```
Comment (abracchi-tw) [inline products/warp-client/src/content/setting-up/linux.md:32]:
```suggestion
The command you need to run is:

```sh
$ warp-cli enable-always-on
```
```
^ please check whether the `$` is correct! if not feel free to adjust

**Review by abracchi-tw — APPROVED** [review]

**Comment by KimJ15** [comment]
@kkrum following up on this PR to see if you need anything else or can it be merged? Thanks!

**Pull request merged**